protocol/p2p/starknet/p2p/proto/event.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+import "p2p/proto/common.proto";
+
+option go_package = "github.com/NethermindEth/juno/p2p/starknet/spec";
+
+message Event {
+    Hash transaction_hash = 1;
+    Felt252 from_address = 3; // looks like mistake?
```

Comment:
the fact that it doesn't have a value for 2?

Reply:
@kirugan, do you know if the value 2 is intentionally missing in the [spec](https://github.com/starknet-io/starknet-p2p-specs/blob/main/p2p/proto/event.proto#L6), or was it overlooked?